src/dnx/servicing.cpp

```diff
@@ -84,15 +107,12 @@ namespace dnx
                 : servicing_root_parent;
 
             auto servicing_root_exists = utils::directory_exists(servicing_root);
-            auto require_servicing = get_require_servicing();
 
             if (!servicing_root_exists)
             {
-                if (require_servicing)
+                if (require_servicing())
                 {
-                    throw std::runtime_error(std::string("Servicing is required for the application to run but the servicing folder '")
-                        .append(dnx::utils::to_string(servicing_root))
-                        .append("' does not exist. The application will now exit."));
+                    throw std::runtime_error("Servicing is required for the application to run but the servicing folder does not exist.");
```

Comment:
Because it is in the exception you never know where it will be displayed. We discussed the same thing with @blowdart a few days ago and ended up removing the path. That's why I did this here too.